lib/ui/page/home/page/chat/message_field/view.dart

```diff
@@ -1020,6 +1116,26 @@ class MessageFieldView extends StatelessWidget {
       ),
     );
   }
+
+  /// Check that [entry] attachment is image or video
```

Comment:
`Check` isn't a verb in third-person form. Effective Dart suggests to documents methods the other way: https://dart.dev/effective-dart/documentation#prefer-starting-comments-of-a-function-or-method-with-third-person-verbs-if-its-main-purpose-is-a-side-effect